manifest.json

```diff
@@ -17,23 +17,29 @@
 
     "content_scripts": [
         {
-            "matches": ["https://*/*/pull/*",
```

Comment:
I made these changes so that it would trigger on organization pages, i.e. `https://github.com/zu/our-repo/pull`, which doesn't match `https://* (github)/* (zu)/pull`.

I'm not sure if it really changed anything, since I think I was actually having problems with the PJAX stuff at that time. Fixing symptoms rather than the problem itself.